src/librustdoc/clean/utils.rs

```diff
@@ -453,3 +453,18 @@ crate fn find_nearest_parent_module(tcx: TyCtxt<'_>, def_id: DefId) -> Option<De
         None
     }
 }
+
+/// Checks that one attribute is `doc`. For example:
```

Comment:
This seems a bit unclear. Maybe something along the lines of

```suggestion
/// Checks for the existence of a `doc(foo)` attribute. For example,
/// checks for the existence of `spotlight` in the attribute below:
```

would be clearer?

Reply:
Sounds good to me.